### DISCUSSION TITLE
What's the validations options do we have?

### DISCUSSION THREAD

**Discussion by Messhias** [goravel/goravel#367] [Q&A] [answered]
I'm looking at something like this: https://laravel.com/docs/10.x/validation#available-validation-rules

**Comment by hwbrzzl** [comment, marked as answer]
Here: https://www.goravel.dev/the-basics/validation.html#available-validation-rules

**Reply by Messhias** [reply]
Thanks, I was looking late at night yesterday, and my attention faded away. Sorry for that and again thanks.
